Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -658,13 +658,7 @@ def perform_rasterize(path: str,
     if browser:
         support_multithreading()
         with ThreadPoolExecutor(max_workers=MAX_CHROME_TABS_COUNT) as executor:
-            demisto.debug(f'path type is: {type(path)}')
-            if type(path) == list:
-                demisto.debug('path type is list')
-                paths = argToList(path)
-            else:
-                demisto.debug('path type is str')
-                paths = [path]
```

Comment:
This is important.
It supports paths that don't start with `http`.
Paths are allowed to contain. a comma.

Reply:
The following code
```python
if type(path) == list:
```

is not working as expected, because the XSOAR never pass a list object as command argument.
When "Is array" is checked on an argument, it will have as effect to not loop on the integration
but the argument will still be a `str` for the command in all cases.
So checking that the type is a list is never true when it's called from a command or a playbook.
The only way to handle list correctly is by calling argToList, I understand your argument that path can contain comma,
but in this case you should pass your argument correctly as a way argToList can handle it like passing the str as a JSON list `["te,st1", "test,2"]`

The old code
```python
paths = [path]
```
is invalid, because passing a list like the XSOAR expect it `test,test2` or `["test1", "test2"]` will just broke as it will ends with a list `["test,test2"] and it's not what we are expecting.

Reply:
You can pass a key from the Context that has a list value.
Then `type(path) == list` will be `True`.

Reply:
I forgot this case, but in that case calling argToList directly is not an issue, as it will just return the list, so everything is fine.

Reply:
Please check what will be the output of rasterizing this URL: https://www.google.com/maps/@48.8588254,2.2644638,12z